admin/tools/docker/4_build-configured-images.sh

```diff
@@ -72,10 +69,9 @@ DOCKERFILE="$DOCKERDIR/Dockerfile"
 cp "$DOCKERDIR/Dockerfile.tpl" "$DOCKERFILE"
 sed -i 's%{{NODE_TYPE_OPT}}%-m%g' "$DOCKERFILE"
 sed -i "s%{{DOCKER_IMAGE_OPT}}%$DOCKER_IMAGE%g" "$DOCKERFILE"
-sed -i "s%{{MASTER_FQDN_OPT}}%${MASTER}%g" "$DOCKERFILE"
 sed -i 's%{{COMMENT_ON_WORKER_OPT}}%%g' "$DOCKERFILE"
 
-TAG="${DOCKER_IMAGE}_master_${MASTER}"
+TAG="${DOCKER_IMAGE}_master"
```

Comment:
The goal of this ticket is to set up master fqdn at container execution so that a given container can be used on different machine. qserv master fqdn is passed via env variable when running "docker run", and qserv-configure is relaunched before starting Qserv.